### ISSUE TITLE
Request: Please import new version on Ansible Galaxy

### ISSUE THREAD

**Issue by bbaassssiiee** [andrewrothstein/ansible-krew#1]
The role doesn't work when referenced in requirements.yml as:

```yaml
roles:
  - src: andrewrothstein.krew
```
Reason: it depends on this role which predates the move of the binaries.
```
- name: andrewrothstein.kubectl
  version: v1.2.3
```

Galaxy only lists v1.1.0 Released 2 years ago

**Comment by bbaassssiiee** [comment]
kubectl_mirror: https://storage.googleapis.com/kubernetes-release/release

**Comment by bbaassssiiee** [comment]
This dependency should be updated.

```yaml
- name: andrewrothstein.kubectl
  version: v1.2.9
```

**Comment by bbaassssiiee** [comment]
Workaround:

```
kubectl_mirror: https://dl.k8s.io/release
kubectl_checksums:
  v1.29.3:
    linux:
      amd64: sha256:89c0435cec75278f84b62b848b8c0d3e15897d6947b6c59a49ddccd93d7312bf
```

**Comment by andrewrothstein** [comment]
I published v1.2.0 to the galaxy delivering krew:0.4.4 and a freshening of the dependencies. Thanks for point this out.
